nautobot/extras/templates/extras/marketplace.html

```diff
@@ -0,0 +1,227 @@
+                                                        <dd>
+                                                            <ul>
+                                                                {% for requirement in app.requires %}
+                                                                    {% for required_app in apps %}
+                                                                        {% if required_app.package_name == requirement %}
+                                                                            {% with required_app_id=required_app.name|slugify %}
+                                                                                <li>
+                                                                                    <a
+                                                                                        data-app="{{ required_app_id }}"
+                                                                                        href="?app={{ required_app_id }}"
+                                                                                    >
+                                                                                        {{ required_app.name }}
+                                                                                    </a>
+                                                                                </li>
+                                                                            {% endwith %}
+                                                                        {% endif %}
+                                                                    {% endfor %}
+                                                                {% endfor %}
+                                                            </ul>
+                                                        </dd>
+                                                    {% endif %}
+
+                                                    {% if app.package_name|default:None != None %}
+                                                        <dt class="text-muted text-uppercase">Package name</dt>
+                                                        <dd><code>{{ app.package_name }}</code></dd>
+                                                    {% endif %}
+                                                </dl>
+                                            </div>
+
+                                            <div>
+                                                <h5>{{ app.headline }}</h5>
+
+                                                <p>{{ app.description }}</p>
+
+                                                {% if app.docs|default:None != None %}
+                                                    <p>
+                                                        <span>
+                                                            Docs:&nbsp;
```

Comment:
Is the `&nbsp;` doing anything here given it's immediately followed by regular whitespace?

Reply:
good point, it should be removed

Reply:
5b683609288f4c4fc97d5eaa28fdb1bf6925ebb6